modules/ocl/cl_image_360_stitch.cpp

```diff
@@ -738,8 +738,11 @@ CLImage360Stitch::sub_handler_execute_done (SmartPtr<CLImageHandler> &handler)
             convert_to_cv_rect (_img_merge_info[i].right, crop_left);
             convert_to_cv_rect (_img_merge_info[idx_next].left, crop_right);
 
```

Comment:
crop_left/crop_right is not useful any more. please remove them.

Reply:
modified function convert_to_cv_rect, please help review.